tests/integration/api/test_event_view_operations.py

```diff
@@ -147,13 +147,12 @@ def test_event_view_ops(event_view, transaction_data_upper_case, source_type):
     columns = [
         col for col in output.columns if not col.startswith("str_") and not col.startswith("dt_")
     ]
-    if source_type == "spark":
+
+    # only snowflake supports timezone in datetime columns
+    if source_type not in ["snowflake"]:
```

Comment:
Is this a change in behaviour for databricks?

Reply:
This is expected since we use cursor description to determine pyarrow type now instead of using the type detected by pandas conversion